Sources/CoreMetrics/Metrics.swift

```diff
@@ -12,26 +12,25 @@
 //
 //===----------------------------------------------------------------------===//
 
-/// A `CounterHandler` represents a backend implementation of a `Counter`.
-///
-/// This type is an implementation detail and should not be used directly, unless implementing your own metrics backend.
-/// To use the SwiftMetrics API, please refer to the documentation of `Counter`.
-///
-/// # Implementation requirements
-///
-/// To implement your own `CounterHandler` you should respect a few requirements that are necessary so applications work
-/// as expected regardless of the selected `CounterHandler` implementation.
-///
-/// - The `CounterHandler` must be a `class`.
-public protocol CounterHandler: AnyObject {
-    /// Increment the counter.
+// MARK: User API
+
+public extension Counter {
```

Comment:
please don't use modifiers in front of `extension`. This modifier propagates to _all_ declarations inside of the extension. In a code review it then frequently becomes impossible to tell if an `internal` or a `public func` has been added. Just remove the `public` here and put it in front of all methods/vars.

Reply:
Okey will do

Reply:
Whoops missed to address this for a long time :O Fixing now